src/opencensus/proto/trace/v1/trace_config.proto

```diff
@@ -23,6 +23,7 @@ option java_outer_classname = "TraceConfigProto";
 option go_package = "github.com/census-instrumentation/opencensus-proto/gen-go/trace/v1";
 
 // Global configuration of the trace service.
+// All maximum values must be specified, or the maximums will be set to zero.
```

Comment:
I would suggest to say:

"All fields must be specified, or the default zero values will be used for each type" - I think proto calls them zero values.

Reply:
How about this?